recipes/papillon/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "papillon" %}
+{% set version = "0.0.8" %}
+{% set sha256 = "7505f977ed053b9eff70ba3a6a0787b227f36c419d8d742e3c74d09ebc7bc9c5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  skip: True  # [py<35]
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python --single-version-externally-managed --record record.txt
+    - setuptools
+    - pandas >=0.17.1
+    - seaborn >=0.8.1
+  run:
+    - python
+    - pandas >=0.17.1
+    - seaborn >=0.8.1
+
+test:
+  imports:
+    - papillon.test
```

Comment:
Probably should just be:

```
- papillon
```

without the `.test`.

Reply:
Actually just now I realized that I got a test folder with some files required for the test, but for a misspelling they are not included in the package. So I don't think is going to work. It will be included in the next build.
Thanks.